contrib/ansible/openshift/register.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+remote=$1
+group=$2
+default_tools_interval=$3
+ose_master_interval=$4
+ose_node_interval=$5
+tool_dir=/var/lib/pbench-agent/tools-default
+ansible_host=$6
+index=$7
+index=$(( index + 1 ))
+label=svt_"$group"_"$index"
+
+## check if a remote is already registered
+for tool in `/bin/ls $tool_dir`; do
+		remote_host=$(echo "$tool" | grep "^remote")
+		remote_hostname=`echo "$remote_host" | awk -F@ '{print $2}'`
+		if [ "$remote" == "$remote_hostname" ]; then
+                	truncate -s -1 $tool_dir/$tool
+                	printf _"$group"_"$index" >> $tool_dir/$tool
+                	label=$(cat "$tool_dir/$tool")       
+	        fi	
+done
+
+## check if nodes are same
+if [[ "$remote" == "$ansible_host" ]]; then
+                exit
+fi
+
+## register tools
+if [ "$group"  == "master" ]; then
+	 pbench-register-tool --label=$label --name=sar --remote $remote
```

Comment:
Consider making the list of commands an external set of parameters to this script so that we don't have the behaviors hard-coded.